### PR TITLE
learning: Added option to specify MSE or SSE as loss function in add_…

### PR DESCRIPTION
**Pull request by dillontsmith**
…backpropagation_learning_pathway. Added SSE as option for loss spec in autodiffcompositions. Default loss function is now MSE in both Composition and AutodiffComposition. Changed expected values of learning-related tests to reflect new defaults.

### CODE REVIEW

**Comment by buildbot-princeton** [comment]
Can one of the admins verify this patch?

**Comment by buildbot-princeton** [comment]
Can one of the admins verify this patch?
